cmake_modules/vcpkgSetup.cmake

```diff
@@ -15,7 +15,9 @@ set (USE_APR OFF)
 set (USE_AZURE OFF)
 set (USE_AWS OFF)
 set (WSSQL_SERVICE OFF)
-set (USE_PYTHON3 OFF)
+set (USE_PYTHON3 ON)
+find_package(Python3 3.6 REQUIRED COMPONENTS Interpreter Development)
```

Comment:
Is 3.6 a minimum version here?

Reply:
I think so - I don't see any diff with or without it.

Reply:
FYI Still fixing the ubuntu build issue.

Reply:
BTW that line comes from the pyembed CMakeLists.txt